pkg/profilecreator/profilecreator.go

```diff
@@ -57,6 +57,16 @@ const (
 	SysInfoFileName = "sysinfo.tgz"
 )
 
+var (
+	// ValidPowerConsumptionModes are a set of valid power consumption modes
+	// default => no args
+	// low-latency => "nmi_watchdog=0", "audit=0",  "mce=off"
+	// ultra-low-latency: low-latency values + "processor.max_cstate=1", "intel_idle.max_cstate=0", "idle=poll"
+	ValidPowerConsumptionModes = []string{"default", "low-latency", "ultra-low-latency"}
```

Comment:
I wonder if we should maintain a "one source of truth" for these settings. tuned has [profiles](https://github.com/redhat-performance/tuned/tree/master/profiles) for various modes but we are still using a somewhat hybrid approach. 
Not for this PR but something we should consider (cc @marcel-apf )

Reply:
Yes, it should be a separate (sub-) package.
Tuned moves data like this in configuration, but since in the container world we will bundle everything in a container image (or even in a binary) anyway, a pkg is good enough IMO.